apps/mull-ui/src/app/components/event-card/event-card.tsx

```diff
@@ -64,22 +64,22 @@ export const EventCard = ({
         </button>
       )}
 
-      <div className="event-card-description" onClick={onClick}>
+      <div className="event-card-description">
         <div className="event-card-text">
           <div className="event-card-title">{event.title}</div>
 
           <div className="event-card-location">{`${distance}km • ${event.location.title}`}</div>
         </div>
         <EventMembers profilePictures={dummyProfilePictures} />
         {/* TODO: Implement share */}
-        <button
+        {/* <button
```

Comment:
Comment for 86.5: Seems like removing the share button broke the placement/styling of the Join event button on the EventCard when user is not joined
![image](https://user-images.githubusercontent.com/33991121/113464417-7e007b80-93fa-11eb-9c83-e81f02a371df.png)

Reply:
It wasn't the share button, but seems like something else caused this, probably the fact I set the background color of all buttons to be white by default